<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARIEL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivos de Imagens</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>651</width>
      <height>291</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>289</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="tw_imgFile">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_addImg">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar Imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_Img">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir Imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_transferImg">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Transferir Imagem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TransferÃªncia de Imagens</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>651</width>
      <height>151</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>149</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="tw_transferImg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>151</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_cancelTransfer">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
